packages/venia-ui/lib/components/CheckoutPage/AddressBook/addressBook.js

```diff
@@ -88,9 +102,8 @@ const AddressBook = props => {
                     onEdit={handleEditAddress}
                 />
             );
-        });
+        }); // Position the default address first in the elements list
```

Comment:
These comments moving around is going to make the readability of the code certainly less.

We should restore them to their original positions